docs/headers.md

```diff
@@ -0,0 +1,52 @@
+# Headers in the Microsoft Graph .NET Client Library
+
+
+The .NET Client Library allows you to add your own custom request headers and inspect the response headers that come back from the Graph service.
+
+## Adding request headers
+
+Custom headers can be added by creating a new header option collection and adding it to the request object:
+
+```chsarp
+List<HeaderOption> options = new List<HeaderOption>();
```

Comment:
Above
"Custom headers can be added by creating a new header option collection"

 change to:

"Custom headers can be added by creating a new option collection"

Below
Let's change this to `List<Option> options = new List<Option>();` so that we can add QueryOption to the collection.